Lib/threading.py

```diff
@@ -689,6 +689,8 @@ def __init__(self, parties, action=None, timeout=None):
         default for all subsequent 'wait()' calls.
 
         """
+        if parties < 1:
+            raise ValueError("parties must be > 0")
```

Comment:
```suggestion
            raise ValueError("parties must be greater than 0")
```

Reply:
Imho this should match the error from asyncio.Barrier

Reply:
Yeah, that's a good idea, I didn't think about that. Ignore this suggestion then, @pygeek